hokusai/commands/diff.py

```diff
@@ -4,7 +4,7 @@
 from hokusai.services.ecr import ECR
 
 @command
-def diff():
+def diff(pr_only=False):
```

Comment:
I tend to leave arguments to the `@command` wrapped functions as positional, (not a keyword with a provided default value as here), because this function should only be called as a direct result of command-line invocation and the Click library ensures this value will always be either `True` or `False` when specified as an option with `is_flag=True` as above.